### DISCUSSION TITLE
How to use functional transforms?

### DISCUSSION THREAD

**Discussion by plutonium-239** [pyg-team/pytorch_geometric#7335] [Q&A] [answered]
I have seen that all transforms have a "functional name" (for ex., [`RandomNodeSplit`](https://pytorch-geometric.readthedocs.io/en/latest/generated/torch_geometric.transforms.RandomNodeSplit.html#torch_geometric.transforms.RandomNodeSplit) is `random_node_split`)
However, I can't figure out how to call this functional.
I've tried:
```py
T.random_node_split(...) # TypeError: 'module' object is not callable
```
I also tried accessing `random_node_split` inside the only other functional module I know, `torch_geometric.nn.functional`, but it does not exist there. Neither does there exist a `functional` module inside `transforms`.

I have looked around but found no questions/docs on this

**Comment by rusty1s** [comment, marked as answer]
The `functional_name` can be used for DataPipe support in PyG. You can find an example [here](https://github.com/pyg-team/pytorch_geometric/blob/master/examples/datapipe.py).
